pioneer/packages/joy-utils/src/transport/proposals.ts

```diff
@@ -187,10 +187,14 @@ export default class ProposalsTransport extends BaseTransport {
   }
 
   async voteByProposalAndMember (proposalId: ProposalId, voterId: MemberId): Promise<VoteKind | null> {
-    const vote = (await this.proposalsEngine.voteExistsByProposalByVoter(proposalId, voterId)) as VoteKind;
-    const hasVoted = (await this.api.query.proposalsEngine.voteExistsByProposalByVoter.size(proposalId, voterId)).toNumber();
+    const votesEntries = await this.api.query.proposalsEngine.voteExistsByProposalByVoter.entries(proposalId);
```

Comment:
I can confirm `.size(..)` is indeed taking an extended amount of time to complete. This seems to only affect double_maps.

The original implementation is using cachedAPI by accessing `this.proposalsEngine` instead of directly `this.api.query.proposalsEngine` would we get some extra speedup in loading if we do this?

```suggestion
    const votesEntries = await this.proposalsEngine.voteExistsByProposalByVoter.entries(proposalId);
```

cc: @Lezek123